README.md

```diff
@@ -157,12 +146,12 @@ If you use PySAL-spaghetti in a scientific publication, we would appreciate usin
 }
 ```
 
-
 ## Citing Work
```

Comment:
Are you trying to keep a list of papers that cite spaghetti here?

Reply:
Yes, this isn't all but some of them.